CONTRIBUTION.md

```diff
@@ -0,0 +1,83 @@
+# Contribution To The Project
+
+Pull Requests and Issues have four possible Labels:
+* WIP - Work In Progress, it is not ready for review/merge
+* Functional - Proposed change changes the test behaviour
+* Metadata - Only Metadata is about to change
+* Scope - Big scope, a commit changes multiple test scenarios
+
+Rules:
+* A commit introduces one logical unit and it should not alter multiple testcases
+* Do not merge your own Pull Request
```

Comment:
I would change it slightly to something like "Do not merge a PR without review". I have seen instances where a PR has been reviewed+approved but not merged, so you might eventually have to merge your own (reviewed/accepted) PR :)

Reply:
Check it now.